doc/5.reference/inlet-outlet-help.pd

```diff
@@ -135,7 +135,7 @@ object to pass arguments to initialize signal inlets in abstractions.
 #X restore 80 259 pd subpatch-with-inlets-and-outlets;
 #N canvas 0 0 600 400 (subpatch) 0;
 #X restore 172 524 pd;
-#X text 70 97 Inlets/outlets are used to receive and get informatin
+#X text 70 97 Inlets/outlets are used to receive and get information
```

Comment:
I was going to open a PR for this one-- would be very nice to fix this since it is discoverable.